lib/ft817/ft817.cpp

```diff
@@ -386,8 +388,7 @@ bool FT817::getKeyer()
 byte FT817::getByte()
 {
 	unsigned long startTime = millis();
-	while (rigCat.available() < 1 && millis() < startTime + 2000) { ; }
-
+	while (rigCat.available() < 1 && millis() < startTime + 2000) { ; }  // I see this came from the VE3BUX lib... but shouldn't something be inside this while{} ? Maybe a timeout?

```

Comment:
Hmmm, this while loop doesn't have anything in it...?  In the VE3BUX code he uses these for some kind of timeout?